apps/core/mix.exs

```diff
@@ -131,7 +131,7 @@ defmodule Core.MixProject do
 
   defp aliases do
     [
-      "ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
+      "ecto.setup": ["ecto.create", "ecto.migrate"],
```

Comment:
the file "priv/repo/seeds.exs" does not exist, I am not sure if you meant for people to have their own `seeds.exs`

but it doesn't work as is